drivers/adc/ad7799/ad7799.h

```diff
@@ -188,8 +190,10 @@ struct ad7799_init_param {
 	uint8_t gain;
 	/** Unipolar/Bipolar Coding */
 	bool polarity;
-	/** Reference Voltage in mV */
-	uint32_t vref_mv;
+	/** Reference Voltage in V */
+	float vref;
+	/** Voltage Scale **/
+	uint32_t scale;
```

Comment:
nitpick: Isn't better an
```
enum ad7799_precision { 
AD7799_PRECISION_UV,
AD7799_PRECISION_MV
}
```
?

Reply:
Or at least a better documentation for the parameter

Reply:
Since the `float` comment is valid, I will use this approach and constrain the precision only to uV and mV.

The `scale` idea came up from the linux IIO framework: `IIO_CHAN_INFO_SCALE`